index.html

```diff
@@ -175,6 +175,19 @@ <h5>Skills</h5>
         </ul>
         </section>
 
-   
+        <section class="contact_form">
```

Comment:
Kindly as requested by the previous reviewer fix the spacing issue between the about and contact sections , you can solve this problem in many ways and I suggest that you avoid setting fixed height for your components the good practice is to use padding and margin properties and let your content size reserve the height from dom flow , add padding-Bottom to the about section and for the color behind the contact form you can add subContainer to this section with properties of contact section and set contact section color to be white , feel free to use other solutions but  please make sure your project looks like the template before re-requesting review .

![image (22)](https://github.com/medijdir/mobileportfolio/assets/59707859/f625f21f-c14e-4eb9-9106-839b6e06773f)